core/node/vm_runner/Cargo.toml

```diff
@@ -16,13 +16,16 @@ zksync_dal.workspace = true
 zksync_contracts.workspace = true
 zksync_state.workspace = true
 zksync_storage.workspace = true
+zksync_core.workspace = true
```

Comment:
Is this a temporary dependency? I'd expect the dependency to be inverted eventually (i.e., core crate to depend on this library).

Reply:
Yeah, so I think `zksync_core` as a crate is going to be gone soon anyway (see https://github.com/matter-labs/zksync-era/pull/1940). Really I just need to depend on state keeper to reuse some of its abstractions. Will change once it has its own crate.

Reply:
The PR is merged now so I changed the dependency to state keeper